README.md

```diff
@@ -26,15 +28,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+there isn't a difference, they do the same thing, however, data-ng just means that the HTML will validate, ng sometimes throws an error
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+in this case, the best usage is going to be A because the best practice is to but it on the html or the body
 ```
 
```

Comment:
The answer is C since a page could have multiple applications, and this is often/usually the case with angular apps.